http-gateway/swagger.yaml

```diff
@@ -132,6 +132,34 @@ paths:
           $ref: '#/components/responses/deviceNotFound'
         500:
           $ref: '#/components/responses/error'
+    delete:
+      tags:
+        - Devices
+      summary: Delete a single resource
+      security:
+        - oauth2:
+          - plgd.devices
+      parameters:
+        - $ref: '#/components/parameters/deviceId'
+        - $ref: '#/components/parameters/resourcePath'
+      description: Delte a single resource
+      responses:
+        200:
+          description: response with content
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResourceRepresentation'
```

Comment:
Is DELETE command seriously returning content of the resource? Is this expected behavior described in the OCF Spec ?

Reply:
Yes it is correct -> it can returns content with error message.